README.md

```diff
@@ -87,6 +103,26 @@ $ yarn install
   $ make test
   ```
 
+  If you receive connection errors similar to the following:
+
+`Failed to open TCP connection to 127.0.0.1:9515 (Too many open files - socket(2) for "127.0.0.1" port 9515)`
+
+  This is due to too many files being opened; the problem may be solved by running the following, prior to running your tests:
+
+
+  ```
+  # One-time usage:
+  ulimit -Sn 65536 && make test
+
+  # To set this permanently, add it to your ~/.zshrc file
+  # or ~/.bash_profile, file depending on your shell:
+  ulimit -Sn 65536
+
+  ```
+
+If you receive errors related to Capybara and feature tests, you may need to install chromedriver. chromedriver can be installed using [Homebrew](https://formulae.brew.sh/cask/chromedriver) or [direct download](https://chromedriver.chromium.org/downloads). The version of chromedriver should correspond to the version of Chrome you have installed; if installing via Homebrew, make sure the versions match up.
```

Comment:
this is the second spot we link to chrome driver downloads, WDYT about using the named link style so that we can use the same URL in both places?

```
[Homebrew][chromedriver-homebrew] or [direct download][chromedriver-direct]

[chromedriver-homebrew]: https://formulae.brew.sh/cask/chromedriver
[chromedriver-direct]: https://chromedriver.chromium.org/downloads

```

Reply:
Creating the Troubleshooting section eliminated this need.